build.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+#! python3
+
+import os, yaml, subprocess
```

Comment:
I would vote for `json`, as most programming languages has good support, both for reading and writing (including JavaScript, weirdly enough).
`yaml`, while a superset of `json` and being much more readable, is not a first-class citizen of most programming languages that I'm aware of.
As for `configparser`, I never really was huge fan of ini files, and different implementations seem to have slight differences in syntax (been a long time since I messed with this, however).

In any case, my vote is for `json`.

Reply:
> including JavaScript, weirdly enough

Just in case you weren't being facetious, JSON is short for JavaScript Object Notation.

Reply:
YAML is staying as this is already in place and is what the upstream build infra uses (from this same file).